test/parallel/test-domain.js

```diff
@@ -163,7 +163,6 @@ expectCaught++;
 // as a callback instead.
 function fn(er) {
   throw new Error('This function should never be called!');
-  process.exit(1);
```

Comment:
Can we ? I have no idea how domain works so I wouldn't know how to modify the test, and it's not really related.